src/main.tsx

```diff
@@ -3,14 +3,20 @@ import ReactDOM from "react-dom/client";
 import {RouterProvider, createBrowserRouter} from "react-router-dom";
 import {Home} from "./pages/home";
 import "./index.css";
+import { Profile } from "./pages/profile";
```

Comment:
import order is the following:
- named_imports
- default_imports
- side_effect_imports (like "./index.css" which applies styles globally)